libnmstate/schema.py

```diff
@@ -278,6 +278,21 @@ class Ethernet:
 
     class SRIOV:
         TOTAL_VFS = "total-vfs"
+        VFS_SUBTREE = "vfs"
+
+        class VFS:
+            ID = "id"
+            MAC = "mac-address"
+            SPOOF_CHECK = "spoof-check"
+            TRUST = "trust"
+            MIN_TX_RATE = "min-tx-rate"
+            MAX_TX_RATE = "max-tx-rate"
+            VLANS_SUBTREE = "vlans"
+
+            class VLANS:
+                ID = "id"
+                PRIORITY = "priority"
+                PROTO = "proto"
```

Comment:
As @cathay4t wrote, `protocol` would be nicer.

Reply:
Done, thanks!